### PR TITLE
blog post: nf-core/fastquorum first release

### PR DESCRIPTION
**Pull request by nh13**
@netlify /blog/2024/fastquorum_intro

### CODE REVIEW

**Comment by nh13** [comment]
I’d like some folks on our team at Fulcrum Genomics to review this before we merge. I’ll make sure to tag them once we figure out who (it’s the long weekend here so thank you for your patience).

**Comment by edmundmiller** [comment]
Sounds good! I'll figure out who else wants to review it on @nf-core/core.

File: src/content/blog/2024/fastquorum_intro.mdx
Comment (jfy133) [inline src/content/blog/2024/fastquorum_intro.mdx:68]:
```suggestion
[Read Structures][read-structure-link] are used to describe how the bases in a sequencing run should be allocated into logical reads.
```

File: src/content/blog/2024/fastquorum_intro.mdx
Comment (jfy133) [inline src/content/blog/2024/fastquorum_intro.mdx:95]: Is it correct that all the kits are not verified? Otherwise seems like an unnecessary column and could just a caveat in italics below the table

File: src/content/blog/2024/fastquorum_intro.mdx
Comment (jfy133) [inline src/content/blog/2024/fastquorum_intro.mdx:119]: Could maybe give 2-3 extra examples of what other diverse things the fgbio tookit has/does (at least as a reader I would be curious, to see if it's worth it before clicking the link for the whole thing)

File: src/content/blog/2024/fastquorum_intro.mdx
Comment (jfy133) [inline src/content/blog/2024/fastquorum_intro.mdx:127]:
```suggestion
Thank you to [James Fellows Yates][james-fellows-yates-link] for these diagrams!
```

File: src/content/blog/2024/fastquorum_intro.mdx
Comment (jfy133) [inline src/content/blog/2024/fastquorum_intro.mdx:144]:
```suggestion
1. For each group of raw reads and calls a consensus sequence, thereby eliminating random errors, significantly improving the accuracy of resulting data. 
    - [`fgbio CallMolecularConsensusReads`][fgbio-callmolecularconsensusreads-link] is used for single-strand UMI schemes, while [`fgbio CallDuplexConsensusReads`][fgbio-callduplexconsensusreads-link] is used for duplex sequencing.
```

File: src/content/blog/2024/fastquorum_intro.mdx
Comment (jfy133) [inline src/content/blog/2024/fastquorum_intro.mdx:182]:
```suggestion
    <YouTube id="https://www.youtube.com/watch?v=qW4kgr3x9Mo" poster='https://img.youtube.com/vi/qW4kgr3x9Mo/maxresdefault.jpg'/>
```

File: src/content/blog/2024/fastquorum_intro.mdx
Comment (FriederikeHanssen) [inline src/content/blog/2024/fastquorum_intro.mdx:4]:
```suggestion
pubDate: 2024-05-27T00:19:00+01:00
```

File: src/content/blog/2024/fastquorum_intro.mdx
Comment (maxulysse) [reply src/content/blog/2024/fastquorum_intro.mdx:182]: That was the main issue for me.
Thanks @jfy133

**Comment by edmundmiller** [comment]
@mashehu or @ewels I'm wondering if we could get the memes and other images to be centered? And maybe make the subway maps larger?

**Comment by mashehu** [comment]
see how james did it with the maintainers minutes post, for how to center and size images

File: src/content/blog/2024/fastquorum_intro.mdx
Comment (nh13) [inline src/content/blog/2024/fastquorum_intro.mdx:148]:
```suggestion
        <Image class="d-block m-auto" src={fastquorum_diagram} alt="Subway diagram of the fastquorum pipeline" width="800" />
```

**Comment by nh13** [comment]
@nf-core-bot fix linting

**Comment by nh13** [comment]
Hey folks, I am at a loss on how to get this valid markdown past all the CI checks, can someone help out?

File: src/content/blog/2024/fastquorum_intro.mdx
Comment (jfy133) [inline src/content/blog/2024/fastquorum_intro.mdx:4]: Should be bumped now

File: src/content/blog/2024/fastquorum_intro.mdx
Comment (jfy133) [inline src/content/blog/2024/fastquorum_intro.mdx:30]: Mightbe worth mentioning here you're talking about genomics/DNA sequencing, given we have increasingly more non-genomics people in nf-core

File: src/content/blog/2024/fastquorum_intro.mdx
Comment (jfy133) [inline src/content/blog/2024/fastquorum_intro.mdx:90]: Should there be a 6 in here, as in the description?

File: src/content/blog/2024/fastquorum_intro.mdx
Comment (jfy133) [reply src/content/blog/2024/fastquorum_intro.mdx:30]: You sort of do this nicely at the 'fgbio: the ...' section

**Review by jfy133 — APPROVED** [review]
A couple of minor comments but content looks good to me (I wonder if  there could be more padding around section headers and figures, but that's one for @mashehu when he's back from leave)

**Comment by nh13** [comment]
I think @cehtolonen from our side wants to make a pass.

**Comment by nh13** [comment]
I got the comments from @cehtolonen live.  Once this pass, we're good on our end.

**Review by FriederikeHanssen — APPROVED** [review]

**Review by edmundmiller — APPROVED** [review]

**Comment by nh13** [comment]
@maxulysse I think you're the remaining reviewer.  Let me know when you think this will go live as we'd like to help extend its reach.

**Review by maxulysse — APPROVED** [review]

**Comment by maxulysse** [comment]
> @maxulysse I think you're the remaining reviewer.  Let me know when you think this will go live as we'd like to help extend its reach.

You're free to merge when you're ready

**Pull request merged**
